api/python/cellxgene_census/tests/experimental/ml/test_pytorch.py

```diff
@@ -262,9 +263,11 @@ def test_encoders(soma_experiment: Experiment) -> None:
 
 
 @pytest.mark.experimental
+# skip pytest if python 3.9
+@pytest.mark.skipif(sys.version_info >= (3, 9), reason="fails intermittently with OOM error for 3.9")
```

Comment:
this "fix"

Reply:
I am fairly certain I have seen it fail on other Python versions.  I'll look for a log to confirm.

Also, the test says `>=3.9`, but the comment says `==3.9`.  Which is correct?